### ISSUE TITLE
Integrated Terminal got smaller 

### ISSUE THREAD

**Issue by demetrisbakas** [Microsoft/vscode#23966]
- VSCode Version: Code 1.11.0 (9122ec4b1068bd391e41974f46a4b54c1b9c90f9, 2017-04-05T20:59:05.375Z)
- OS Version: Windows_NT ia32 10.0.14393
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|code-gnu-global|austin|0.2.1|
|gitignore|codezombiech|0.5.0|
|python|donjayamanne|0.6.0|
|cpptools|ms-vscode|0.10.5|
|csharp|ms-vscode|1.8.1|
|prolog|rebornix|0.0.3|
|vscode-icons|robertohuertasm|7.5.1|
|code-spell-checker|streetsidesoftware|1.0.2|
|vim|vscodevim|0.6.13|;

---

Steps to Reproduce:

1. Press Ctrl+`

Basically the problem is that the terminal takes only 2/3 of the screen lengthwise and much space is wasted. I have no idea whats causing this and this problem didn't appear in any of the older versions. 
I use the bash for windows as the default terminal if that helps in any way.

**Comment by chrmarti** [comment]
@demetrisbakas Lengthwise meaning vertically?

**Comment by demetrisbakas** [comment]
No, horizontally.

**Comment by chrmarti** [comment]
@demetrisbakas Could you attach a screenshot?

**Comment by Tyriar** [comment]
This is a temporary workaround to minimize the impact of a bug on Windows https://github.com/Microsoft/vscode/issues/19665, it should hopefully be back to normal in the next 1 or 2 versions.

**Comment by demetrisbakas** [comment]
@chrmarti 
![vs code terminal](https://cloud.githubusercontent.com/assets/10773045/24734870/8fff2f4c-1a88-11e7-8111-d9d448556e57.png)

**Comment by Tyriar** [comment]
Here's the comment on the change I made, capping it at 159 characters in width https://github.com/Microsoft/vscode/issues/19665#issuecomment-285975836

**Comment by demetrisbakas** [comment]
@Tyriar I cant find the file "terminalInstance.ts" nowhere on Windows.

**Comment by Tyriar** [comment]
@demetrisbakas this was a temporary change in the latest version of VS Code to try to workaround a much worse issue on Windows.
